thirdparty/scripts/build_credis.sh

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
 
+# Usage: by default jemalloc is used; however, if tcmalloc is installed on the
+# system, credis' CMakeLists.txt will prefer it over jemalloc.  To avoid build
+# failures use:
+#
+#     CREDIS_TCMALLOC=1 build_credis.sh
```

Comment:
If this is the case, then why don't we just call it this way in https://github.com/ray-project/ray/blob/470887c2ad62b0ab30b41e2fe2db8569fc81a879/thirdparty/scripts/setup.sh#L31

Reply:
In short many default instances/platforms don't have tcmalloc.  This note is just for adventurous usage only; the default cases will work without this flag.